robottelo/ui/base.py

```diff
@@ -471,3 +473,25 @@ def click(self, locator, wait_for_ajax=True,
         element.click()
         if wait_for_ajax:
             self.wait_for_ajax(ajax_timeout)
+
+    def select(self, locator, value, wait_for_ajax=True, timeout=30):
+        """Select the element described by the ``locator``.
+
+        :param locator: The locator that describes the element.
+        :oaram value: The value to select from the dropdown
+        :param wait_for_ajax: Flag that indicates if should wait for AJAX after
+            clicking on the element
+        :param timeout: The amount of time that wait_fox_ajax should wait. This
+            will have effect if ``wait_fox_ajax`` parameter is ``True``.
+        :raise: UINoSuchElementError if the element could not be found.
+
+        """
+        element = self.wait_until_element(locator)
+        if element is None:
+            raise UINoSuchElementError(
+                '{}: element with locator {} not found while trying to click.'
```

Comment:
ah!, I had many issues while re-factoring this, had to drop and add code back , may be in that process missed it. :)